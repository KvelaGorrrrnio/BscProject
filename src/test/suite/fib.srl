// Program for computing the n'th fibonacci pair
int n
int w
int v
n ^= 16
w ^= 1
from (v = 0) do
  v += w
  swap v w
  n -= 1
until (n <= 1)
