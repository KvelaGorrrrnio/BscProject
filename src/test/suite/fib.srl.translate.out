int n
int v
int w

init: entry
  n ^= 16
  w ^= 1
goto loop1

loop1: fi (v = 0) init loop1
  v += w
  swap v w
  n -= 1
if (n <= 1) contloop1 loop1

contloop1: from loop1
  skip
exit
